plugins/chat/config/routes.rb

```diff
@@ -29,6 +29,8 @@
     get "/mentions/groups" => "hints#check_group_mentions", :format => :json
 
     get "/channels/:channel_id/threads/:thread_id" => "channel_threads#show"
+
+    delete "/messages/:message_id" => "messages#delete"
```

Comment:
Not sure if we want all these API endpoints to start with channels and include the channel ID? If so I can just change it.

Reply:
Yes I prefer we follow this scheme, it ensures we have a channel ID, and we should always check our message is part of this channel, it's one of the few super important pillars of chat.

Reply:
Sure no problem, will change